master/internal/agent/proto.go

```diff
@@ -17,6 +17,7 @@ func ToProtoAgent(a AgentSummary) *proto.Agent {
 		Slots:          slots,
 		Containers:     nil,
 		Label:          a.Label,
+		ResourcePool: 	a.ResourcePool,
```

Comment:
q: do you have the change to `summaries.go`? I didn't find the change to `newAgentSummary`.

Reply:
No, I didn't add resource pool to that because it didn't look like that field would be useful - AFAICT `agentSummary` is only used in scheduling/provisioning which all happens within a single resource pool and the summaries are not user-facing.

Reply:
> agentSummary is only used in scheduling/provisioning which all happens within a single resource pool and the summaries are not user-facing.

I see where my confusion came from. There are two `AgentSummary`s defined both in `sproto` and `agent`.